scripts/utils/create_package_template.py

```diff
@@ -419,6 +419,17 @@ def get_script_directory():
 }
 
 
+def set_placeholder_data(args):
+    for arg in TYPES.get(args.type)["arguments"]:
+        if not arg:
+            args.__setattr__(arg, f"<{arg}>")
+        if arg == "pkg_name":
+            # default placeholder package name
+            args.__setattr__("pkg_name", "new-package")
+    sys.exit()
+    return args
```

Comment:
Do we want to exit here?

Reply:
Nope, good catch. Forgot to push the last commit. Will add more documentation on this PR now.